charters/wot-wg-2023-draft.html

```diff
@@ -242,24 +242,39 @@ <h3>
-              <p class="milestone"><b>Expected completion:</b> <i class="todo">[Q1–4 yyyy]</i></p>
-              <p class="todo">Per <a href="https://www.w3.org/Consortium/Process/#WGCharter">5.2.6</a>, for every Recommendation Track deliverable that continues work on a Working Draft (WD) published under any other Charter (including a predecessor group of the same name), for which there is an existing <a href="https://www.w3.org/Consortium/Process/#exclusion-draft">Exclusion Draft</a>: </p>
-              <p><b>Adopted Draft:</b> The <span class="todo">title, stable URL, and publication date of the <a href="https://www.w3.org/Consortium/Process/#adopted-draft">Adopted Draft</a></span> which will serve as the basis for work on the deliverable.
-              </p><p><b>Exclusion Draft:</b> The <span class="todo">title, stable URL, and publication date of the most recent <a href="https://www.w3.org/Consortium/Process/#exclusion-draft">Exclusion Draft</a></span>.
-              <span class="todo">Exclusion period <b>began</b>; Exclusion period <b>ended</b>.</span> <span class="todo">(this <a href="https://www.w3.org/2004/01/pp-impl/charter-assistant?wgid=32061">charter assistant</a> helps in producing the list. use the proper <a href="https://www.w3.org/2004/01/pp-impl/">wgid</a>)</span>
-              </p><p><b>Exclusion Draft Charter:</b> The  <span class="todo">stable URL of the <a href="https://www.w3.org/Consortium/Process/#other-charter">Working Group charter</a></span> under which the most recent Exclusion Draft was published.
+              <p>This specification defines a Discovery process for WoT, and would be a backward-compatible
+                 update to the existing WoT Discovery Specification.</p>
+
+              <p class="draft-status"><b>Draft state:</b>No draft</p>
+              <p class="milestone"><b>Expected completion:</b> Q3 2024</p>
+              <p><b>Adopted Draft:</b>
+              This will be a revision of the current <i>Web of Things (WoT) Discovery</i> specification published
+              at <a href="https://www.w3.org/TR/wot-discovery/">https://www.w3.org/TR/wot-discovery/</a>,
+              as a Candidate Recommendation on 19 January 2023.</p>
+              </p>
+            </p></dd>
+            <dt id="security-deliverable" class="spec">Web of Things (WoT) Security and Privacy</dt>
+            <dd>
+              <p>This specification defines the overall security and privacy constraints, policies,
+                 and vocabulary for WoT systems.  It would be a new specification consolidating
+                 normative material that currently is scattered in several other WoT specifications.</p>
```

Comment:
My opinion is that there should not be a separate normative specification for security and privacy.

There are currently four normative specifications which implementers need to read in order to implement a fully featured, W3C conformant, WoT Consumer or Thing, which in my view is already too many.

Security is never implemented on its own, it is always securing something else. My recommendation would be to keep any security & privacy related normative assertions in the existing normative specifications (including the required privacy & security sections), in order to aid readability.

I can see why collecting all security & privacy related material in one place might make things easier for editors, but I would argue it makes things harder for readers/implementers of the specifications if they have to cross-reference multiple documents.

Reply:
reorganized, no longer a separate security deliverable